Jenkinsfile

```diff
@@ -191,7 +191,7 @@ def runCypressE2e(){
           repo="maven {url '"$M2_LOCAL_REPO"'}"
           sed -i "/repositories {/a$repo" hc-qa-project/build.gradle
           chmod +x setup.sh;
-          ./setup.sh dhs=false mlHost=localhost;
+          ./setup.sh dhs=false mlHost=localhost mlAdminUsername=admin mlAdminPassword=admin;
```

Comment:
Use mlSecurityUsername/Password instead - see https://github.com/marklogic-community/ml-gradle/wiki/Property-reference#manage-server-connection-properties

Reply:
Can we use mlSecurityUsername on DHS too?

Reply:
Thanks @rjrudin, modified the parameter names